src/pages/PromotionsPage/index.jsx

```diff
@@ -2,52 +2,77 @@ import React from 'react';
 import { FormattedMessage, injectIntl } from 'react-intl';
 import { useHistory } from 'react-router-dom';
 import './style.scss';
+import {loadStripe} from '@stripe/stripe-js';
+import {Elements} from '@stripe/react-stripe-js';
+import CheckoutForm from '../../components/Payment';
+
 
 const endPointUrl = process.env.REACT_APP_API_URL;
 
+const stripePromise = loadStripe(process.env.REACT_APP_STRIPE_PUBLISHABLE_KEY);
+
 const PromotionsPage = props => {
     const history = useHistory();
-    const business = props.id || props.match.params.id;
-
-    const goToPayment = (clicks) => {
-        // take user to payment
-        // object:
-        // items: [arrayOfItems]
-        // currency: >
-        // 10: Bronze
-        // 100: Silver
-        // 1000: Gold
-    };
+    const business = props.match.params.id || props.id;
+    const [selected, setSelected] = React.useState(100);
```

Comment:
change this variable name to `selectedPromotion` and have equal to an object that looks like this by default `{ amount: null, clicks: 0}`